File: src/main/java/com/trade/securities/domain/MarketData.java
package com.trade.securities.domain;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class MarketData {
    private String basDt; // Base Date
    private String idxNm; // Index Name
    private String clpr; // Closing Price
    private String fltRt; // Fluctuation Rate
    // Add more fields as needed based on API response
}

File: src/main/kotlin/com/trade/securities/domain/MarketData.kt
package com.trade.securities.domain

data class MarketData(
    val basDt: String? = null, // Base Date
    val idxNm: String? = null, // Index Name
    val clpr: String? = null, // Closing Price
    val fltRt: String? = null  // Fluctuation Rate
)
